modules/apps/transfer/ibc_module_test.go

```diff
@@ -89,6 +96,7 @@ func (suite *TransferTestSuite) TestOnChanOpenInit() {
 				ConnectionHops: []string{path.EndpointA.ConnectionID},
 				Version:        types.Version,
 			}
+			v1 = false
```

Comment:
In theory you shouldn't need this since the zero value for uninitialised boolean variables is already false.